tools/cmake/SetLinkerScript.cmake

```diff
@@ -0,0 +1,48 @@
+#
```

Comment:
add license info

Reply:
Fixed in [this force push](https://github.com/ARMmbed/mbed-os/compare/99614a04adb48aa444af67bea3a40d6f42a0ac58..7e50d55f8c01fdcd2735eb36d4600cddbbe17d88)